CONTRIBUTING.md

```diff
@@ -288,10 +288,6 @@ Before a PR can be merged, it should satisfy the following:
 
 ### Release Checklist
 
-- [ ] Update the GLVis version in the following files:
-    - [ ] `CHANGELOG`
-    - [ ] `makefile`
-    - [ ] `CMakeLists.txt`
```

Comment:
Why remove these?

Reply:
There is no version actually in `makefile` or `CMakeLists.txt`

Reply:
And `CHANGELOG`?

Reply:
I thought that was obvious, but I can bring that one back if you prefer

Reply:
It's better to have this list -- I see version numbers in other places that we should add: e.g. `share/Info.cmake.plist.in` and `share/Info.plist`. There maybe other places too.

Reply:
ok

Reply:
Another file that has version: `vcpkg.json`.

Reply:
Done in https://github.com/GLVis/glvis/pull/227/commits/17e2a3cd6ea843ed9d7bd8213c63ce16d942345d